jupyterlab/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling>=1.4.0", "jupyterlab==4.0.0", "hatch-nodejs-version"]
+requires = ["hatchling<=1.18.0", "jupyterlab==4.0.0", "hatch-nodejs-version"]
```

Comment:
Could you add a comment with a link to the issue you're working around?

(I'm trying to avoid a future like [this](https://github.com/microsoft/iqsharp/blob/main/src/Python/requirements.txt) where you can never be sure exactly what you're breaking by unpinning a dependency)